test/library/standard/FileSystem/mkdir/illegalArgs.good

```diff
@@ -0,0 +1,2 @@
+FileNotFoundError: No such file or directory (mkdir called with illegal path: '')
```

Comment:
Does this .good file need to be updated to say "SystemError"?

Reply:
Nope. The `ioerror(ENOENT:syserr)` throws a `FileNotFoundError`, which is just a subtype of `SystemError`.